### PR TITLE
Add support for vendor_boot pack/unpack

### PR DESCRIPTION
**Pull request by HighwayStar**
Tested to work with no issue unpack/pack on amlogic android 11 roms

### CODE REVIEW

**Comment by althafvly** [comment]
@HighwayStar add it in rkunpack.sh,rkpack.sh and awpack.sh aswell. vendor_boot and vendor_boot_a

**Comment by HighwayStar** [comment]
updated, also added vendopr_boot and vendor_boot_a variants and odm_ext, which exist in aml a11 roms too

**Comment by HighwayStar** [comment]
And added vbmeta_system partition to scripts

File: rkunpack.sh [hunk 1]
```diff
@@ -123,7 +123,7 @@ elif [ $level = 3 ]; then
         mkdir level3
     fi
 
-    for part in boot recovery boot_a recovery_a; do
+    for part in boot recovery vendor_boot  boot_a recovery_a vendor_boot_a; do
```
Comment (althafvly) [inline rkunpack.sh]: remove that extra space.

File: dump_to_aml.bat [hunk 2]
```diff
@@ -52,7 +52,7 @@ md level1 level2\config level3
 
 if not exist out md out
 
-FOR %%A IN (boot recovery logo dtbo vbmeta bootloader odm oem product vendor system system_ext) DO (
+FOR %%A IN (boot recovery vendor_boot logo dtbo vbmeta bootloader odm oem product vendor system system_ext odm_ext) DO (
```
Comment (althafvly) [inline dump_to_aml.bat]: i dont maintain windows script now, so not really needed.

Comment (HighwayStar) [reply dump_to_aml.bat]: Fixed that and added some more code dtb related

File: pack.sh [hunk 3]
```diff
@@ -199,6 +199,18 @@ elif [ $level = 3 ]; then
...
+            cp level1/_aml_dtb.PARTITION level3/$part
+        fi
+    done
+
+    for part in "boot_a/split_img/boot_a.PARTITION-dtb" "boot_a/split_img/boot_a.PARTITION-second" "recovery_a/split_img/recovery_a.PARTITION-dtb" "recovery_a/split_img/recovery_a.PARTITION-second" "vendor_boot_a/split_img/vendor_boot_a.PARTITION-dtb" "vendor_boot_a/split_img/vendor_boot_a.PARTITION-second"; do
+        if [ -f level3/$part ]; then
+            cp level1/_aml_dtb.PARTITION level3/$part
+        fi
+    done
+
```
Comment (althafvly) [inline pack.sh]: This would copy dtb before its packed, means old one.

Comment (HighwayStar) [reply pack.sh]: But new dtb just generated above by dtbTool and placed to level1/_aml_dtb.PARTITION

Comment (althafvly) [reply pack.sh]: it might also need to be gzipped if size is an issue.  Code below.

Comment (HighwayStar) [reply pack.sh]: Moved it back, dont have device with big dtb that need compression to test.

**Pull request merged**